packages/augur-ui/src/modules/create-market/components/create-market-view/create-market-view.tsx

```diff
@@ -9,9 +8,23 @@ import Landing from "modules/create-market/containers/landing";
 import Styles from "modules/create-market/components/create-market-view/create-market-view.styles.less";
 
 interface CreateMarketViewProps {
+  isMobileSmall: Boolean;
+  currentTimestamp: number;
+  gasPrice: number;
+  history: Object;
```

Comment:
Move the HistoryObject interface to types.ts and reference it here and not Object.